src/site/content/en/learn-core-web-vitals/index.njk

```diff
@@ -1,6 +1,56 @@
 ---
-layout: path
+layout: collection
 override:tags: []
 date: 2020-09-16
 pathName: learn-core-web-vitals
+cards:
+  - title: Thinking on ways to solve cafe wall illusion
+    description: Praesent accumsan eros orci quis congue metus porta a sed dapibus magna nunc non finibus erat ullamcorper sit amet.
+    thumbnail: image/SZHNhsfjU9RbCestTGZU6N7JEWs1/MqeKXDtUPO7zszzpRXcF.png
+    eyebrow:
+      icon: featured
+      text: Featured
+    cssClass: 'card-vertical'
+  - title: 'The CSS Podcast: Media query range syntax'
+    description: Praesent accumsan eros orci quis congue metus porta a sed dapibus magna.
+    thumbnail: image/SZHNhsfjU9RbCestTGZU6N7JEWs1/VwL892KEz6bakZMlq10D.png
+    eyebrow:
+      icon: podcast
+      text: Podcast
+    cssClass: 'card-horizontal col-2 bg-tertiary'
```

Comment:
We cannot expect authors to know internal css class names. Let's introduce 2-4 types of cards and assign a cardType to each. This should be documented for authors in the docs page.

Reply:
I added these YAML properties to allows the content creator to determine a card layout
- `cardLayout` 
- `theme`
- `column` 
- `row`